commands/pexpireat.md

```diff
@@ -1,12 +1,24 @@
 `PEXPIREAT` has the same effect and semantic as `EXPIREAT`, but the Unix time at
 which the key will expire is specified in milliseconds instead of seconds.
 
+## Options
+
+The `PEXPIREAT` command supports a set of options since Redis 7.0:
+
+* `NX` -- Set expiry only when the key has no expiry
+* `XX` -- Set expiry only when the key has an existing expiry
+* `GT` -- Set expiry only when the new expiry is greater than current one
+* `LT` -- Set expiry only when the new expiry is less than current one
+
+A non-volatile key is treated as an infinite TTL for the purpose of `GT` and `LT`.
+The `GT`, `LT` and `NX` options are mutually exclusive.
+
 @return
 
 @integer-reply, specifically:
 
 * `1` if the timeout was set.
-* `0` if `key` does not exist.
+* `0` if `key` does not exist or the expiry was not applied due to provided option.
```

Comment:
Would this be clearer as?
```suggestion
* `0` if the timeout was not set.
```

I don't know if we need to enumerate the different ways that can happen.

Reply:
i think it can save some confusion.
maybe we want to split it to format part and then examples that make it clear? like:
```suggestion
- * `0` if `key` does not exist or the expiry was not applied due to provided option.
+ * `0` if the timeout was not set. e.g. key doesn't exist, or operation skipped due to the provided arguments.
```

Reply:
I would recommend to provide such examples for why timeout could not be set.